.rubocop.yml

```diff
@@ -17,3 +17,12 @@ ClassLength:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Metrics/BlockLength:
+  CountComments: false  # count full line comments?
+  Max: 25
```

Comment:
This is a new metric in rubocop and I'm mostly just telling it to ignore the normal stuff we ignore plus the `test_helper` which has a huge block that is really not worth rewriting at this time.